pkg/rtc/room.go

```diff
@@ -505,6 +505,34 @@ func (r *Room) OnMetadataUpdate(f func(metadata string)) {
 	r.onMetadataUpdate = f
 }
 
+func (r *Room) SimulateScenario(participant types.Participant, simulateScenario *livekit.SimulateScenario) error {
+	switch scenario := simulateScenario.Scenario.(type) {
+	case *livekit.SimulateScenario_SpeakerUpdate:
+		r.Logger.Infow("simulating speaker update", "participant", participant.Identity())
+		go func() {
+			<-time.After(time.Duration(scenario.SpeakerUpdate) * time.Second)
```

Comment:
Should we think about parameterising scenarios? For example, the duration here?

Reply:
Do you mean a separate duration field? And leaving `SpeakerUpdate` as bool?

Reply:
Sorry, I was reading things wrong. My bad.